src/common/index.ts

```diff
@@ -2,13 +2,36 @@ import * as constants from './constants'
 import * as errors from './errors'
 import * as validate from './validate'
 import * as serverInfo from './serverinfo'
+import {xAddressToClassicAddress} from 'ripple-address-codec'
+
+export function ensureClassicAddress(account: string | undefined): string {
```

Comment:
I've cleaned this up so that we don't check for `undefined` here. The issue with the recommendation is that it would decode the X-address four times (see [the implementation of `isValidXAddress` here](https://github.com/ripple/ripple-address-codec/blob/master/src/index.ts#L112)) when it only needs to be done once.